packages/block-library/src/list/v2/edit.js

```diff
@@ -126,16 +127,22 @@ function IndentUI( { clientId } ) {
 	);
 }
 
-function Edit( { attributes, setAttributes, clientId } ) {
-	const blockProps = useBlockProps();
+function Edit( { attributes, setAttributes, clientId, style } ) {
+	const blockProps = useBlockProps( {
+		...( Platform.isNative && { style } ),
```

Comment:
What is this used for?

Reply:
This is used to inherit the styles, mostly used for the color text/background customization and global styles.